dashboard/app/views/followers/student_user_new.html.haml

```diff
@@ -41,46 +44,47 @@
           = f.hidden_field :hashed_email
 
           .itemblock
-            .labelblock
-              = t('activerecord.attributes.user.name')
+            = f.label :name, t('activerecord.attributes.user.name')
             = f.text_field :name, class: 'input-xlarge fieldblock', autofocus: true
           %br/
           .itemblock
-            .labelblock
-              = t('activerecord.attributes.user.email')
+            = f.label :email, t('activerecord.attributes.user.email')
             = f.email_field :email, class: 'input-xlarge fieldblock'
           %br/
           - if f.object.password_required?
             .itemblock
-              .labelblock
-                = t('activerecord.attributes.user.password')
+              = f.label :password, t('activerecord.attributes.user.password')
               = f.password_field :password, class: 'input-xlarge fieldblock'
               %span#password_message
             %br/
             .itemblock
-              .labelblock
-                = t('activerecord.attributes.user.password_confirmation')
+              = f.label :password_confirmation, t('activerecord.attributes.user.password_confirmation')
               = f.password_field :password_confirmation, class: 'input-xlarge fieldblock'
             %span#password_message_confirmation
             %br/
             - if user_type == User::TYPE_TEACHER
               .itemblock
-                .labelblock
-                  = t('signup_form.school_name')
+                = f.label :school, t('signup_form.school_name')
                 = f.text_field :school, class: 'input-xlarge fieldblock'
               %br/
               .itemblock
-                .labelblock
-                  = t('signup_form.school_address')
+                = f.label :full_address, t('signup_form.school_address')
                 = f.text_area :full_address, class: 'input-xxlarge fieldblock', placeholder: t('signup_form.address_placeholder'), rows: 4, maxlength: 255
               %br/
 
               = f.hidden_field :age, value: '21'
             - if user_type == User::TYPE_STUDENT
               .itemblock
-                .labelblock
-                  = t('signup_form.age')
+                = f.label :age, t('signup_form.age')
                 = f.select :age, User::AGE_DROPDOWN_OPTIONS, include_blank: true, class: 'fieldblock'
+              %br/
```

Comment:
can you add a before/after screenshot to make sure this <br> blends in?

Reply:
Done. I understand your concern, it looks like I'm adding rogue breaks. But the styling on this page was actually broken before I made any changes to it. See the screenshots.